### ISSUE TITLE
add more/better logging to help ease debugging

### ISSUE THREAD

**Issue by stephenmathieson** [flickr/yakbak#31]
I'm currently debugging some tests that fail intermittently. The tests hit `yakbak` for each request and only _sometimes_ have a matching tape. it'd be extremely useful if setting `DEBUG=yakbak*` when running my test suite would output more than this:

```
  yakbak:server req +0ms /some/path?with=some&query=strings
```

would you take a PR that adds a few more `debug()`s for things like:

- the request path
- the request hash
- if a tape was found
- if yakbak will respond to the request

**Comment by jeremyruppel** [comment]
Absolutely, I'm all for more helpful debug logs. Send a PR over and I'll ship it as soon as possible.
